tests/libvmifact/architecture.go

```diff
@@ -19,12 +19,51 @@
 
 package libvmifact
 
-var architecture string
+import "kubevirt.io/kubevirt/pkg/libvmi"
+
+const (
+	// required to start qemu on ARM with UEFI firmware
+	// https://github.com/kubevirt/kubevirt/pull/11366#issuecomment-1970247448
+	armMinimalBootableMemory = "128Mi"
+
+	defaultMinimalBootableMemory = "1Mi"
+)
+
+var (
+	architecture string
+	archAware    architectureAwareMemoryProvider = defaultArchMemoryProvider{}
+)
 
 func RegisterArchitecture(arch string) {
 	architecture = arch
+	if architecture == "arm64" {
```

Comment:
What is the benefit of architecture variable? It's not used outside this function, right? If so, we can just then check it by using `arch` argument, e.g. `if arch == "arm64"`?

I have also a few questions:

1. Is there chance that the value of arch can be arm32 or aarch32?
2. ARM is also known as aarch64, can we get it and not arm64?

If the answer is yes, then let's use enum?

Reply:
1. even if we do, this is not a change from the current behavior
2. again, not a change from the current behavior